partner_invoicing_mode/models/account_invoice.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    def _validate_invoice(self):
+        return self.sudo().action_post()
```

Comment:
I'm not sure I understand why this method is needed.
Can't we call/delay `action_post` directly ?